src/entryPoints/AuthModule/TwoFaAuth/TwoFaAuth.res

```diff
@@ -147,7 +147,7 @@ let make = (~setAuthStatus, ~authType, ~setAuthType) => {
     try {
       open CommonAuthUtils
       let valuesDict = values->getDictFromJsonObject
-      let email = valuesDict->getString("email", "")
+      let email = valuesDict->getString("email", "")->String.toLowerCase
```

Comment:
this only changes the email that is sent in api payload, can you make the same change for email in `emailField` so that it is consistent

Reply:
If we convert to lowercase directly in emailField, it would prevent mixedcase input at all, which isn’t ideal. Typically, sites allow any case in the field and only convert to lowercase when sending to the backend. Could you confirm if this is correct?

Reply:
As discussed with @muditbhutani , currently we will make this change from frontend, later it should be handled in backend itself to avoid inconsistency in frontend